libs/screen/image.d.ts

```diff
@@ -110,6 +150,19 @@ interface Image {
 
     //% shim=ImageMethods::revision
     revision(): number;
+
+    /**
+     * Draw given image with transparent background on the current image
+     */
+    //% shim=ImageMethods::drawTransparentImage
+    //% blockNamespace="images"
+    //% group="Drawing"
+    //% block="$this draw $from at x $x y $y"
+    //% this.shadow=variables_get
+    //% this.defl="picture"
+    //% from.shadow=screen_image_picker
+    //% weight=0
+    drawTransparentImage(from: Image, x: int32, y: int32): void;
```

Comment:
Does this need a blockId?

Reply:
Do you want to support scale? :)

Reply:
Added the blockid! Not opposed to supporting scale, but it might be better off in the scale extension?

Also, fun fact: none of the blocks in the image category other than the ones in this pr have block ids 😅